extras/local-omi-bt/connect-omi.py

```
@@ -76,15 +62,78 @@ def handle_ble_data(sender: Any, data: bytes) -> None:
+
+    def handle_button_event(sender: Any, data: bytes) -> None:
+        try:
+            state = parse_button_event(data)
+        except Exception as e:
+            logger.error("Button event parse error: %s", e)
+            return
+        if state != ButtonState.IDLE:
+            logger.info("Button event: %s", state.name)
+            try:
+                loop = asyncio.get_running_loop()
+                loop.create_task(send_button_event(state.name))
```

Ruff (0.14.14)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)